prime-dotnet-webapi/Services/BannerService.cs

```diff
@@ -67,6 +67,8 @@ public async Task<IEnumerable<BannerViewModel>> GetBannersAsync(BannerLocationCo
                 banner.EndTimestamp = DateTime.SpecifyKind(banner.EndTimestamp, DateTimeKind.Utc);
             });
 
+            banners.Reverse();
```

Comment:
I would look at EF Core's query API and use `OrderByDescending(...)`.  You should be able to do all the work in the DB query.

Reply:
It seems that we don't have an attribute for when the banner was created in order to sort using the `OrderByDescending()`
The closest I can find to sort them based on when they were posted would be the `Banner.Id`. We have StartTime  EndTime attributes but those are irrelevant to the posting time of the banner.

Reply:
That's what I would have done since don't have a CreatedDate or UpdatedDate to adjust the resulting set of banners.

Reply:
And they haven't asked to have banners change order based on changes/updates